src/cache/cache_las.c

```diff
@@ -161,7 +163,7 @@ __wt_las_cursor_create(WT_SESSION_IMPL *session, WT_CURSOR **cursorp)
 	 * opens (the first update is safe because it's single-threaded from
 	 * wiredtiger_open).
 	 */
-	btree = S2BT(session);
+	btree = ((WT_CURSOR_BTREE *)(*cursorp))->btree;
```

Comment:
This change appears a few times, is it a consequence of no longer saving and restoring the ``session_dhandle`` in ``__meta_btree_apply``? I'd rather keep the ``S2BT`` semantic if possible.